src/scatra_ele/4C_scatra_ele.cpp

```diff
@@ -93,147 +93,147 @@ void Discret::Elements::TransportType::setup_element_definition(
   std::map<std::string, Input::LineDefinition>& defs = definitions["TRANSP"];
 
   defs["HEX8"] = Input::LineDefinition::Builder()
-                     .add_int_vector("HEX8", 8)
+                     .add_named_int_vector("HEX8", 8)
```

Comment:
The problematic cases were these: previously, we added the "HEX8" tag on a higher level. Now, it is required here.
We actually need to look at the "HEX8" tag quite early; at least, this is how element input is currently implemented. Now I "peek" at the cell type tag...

Reply:
ah ok, I never looked into these internals of the element reader. I thought that HEX8 was already a named integer vector.